packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
+    border: 1px solid #e0e8ed;
+    color: #012159;
+    label {
+      span {
+        display: block;
+        padding: 10px;
+        margin-bottom: 1px;
+        &:hover {
+          background: #e9f8ff;
+        }
+      }
+      input {
+        display: none;
+        &:checked + span {
+          background: #e9f8ff;
+        }
+      }
+    }
+  }
+  .transfer-buttons {
+    width: 10%;
```

Comment:
Can we make this a fixed width when we change the transfer to flexbox? Will mean that on smaller screen sizes the buttons don't get crushed

Reply:
Fair. Done.